rules/windows/process_creation/win_susp_sysvol_access.yml

```diff
@@ -17,7 +17,7 @@ logsource:
     product: windows
 detection:
     selection:
-        CommandLine: '*\SYSVOL\\*\policies\\*'
+        CommandLine|contains: '\SYSVOL\\*\policies\\'
```

Comment:
could you please use the following construction:

```
        CommandLine|contains|all: 
            - '\SYSVOL\'
            - '\policies\'
```

Reply:
Applied